SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -544,28 +544,134 @@ void CFlowOutput::SetAnalyzeSurface(const CSolver* const*solver, const CGeometry
   for (unsigned short iVar = 0; iVar < nSpecies; iVar++)
     SetHistoryOutputValue("SURFACE_SPECIES_" + std::to_string(iVar), Tot_Surface_Species[iVar]);
 
+  if (config->GetKind_Species_Model() != SPECIES_MODEL::NONE)
+    SetAnalyzeSurface_SpeciesVariance(solver, geometry, config, Surface_Species_Total, Surface_MassFlow_Abs_Total,
+                                      Surface_Area_Total);
+
+  if ((rank == MASTER_NODE) && !config->GetDiscrete_Adjoint() && output) {
+
+    cout.precision(6);
+    cout.setf(ios::scientific, ios::floatfield);
+    cout << endl << "Computing surface mean values." << endl << endl;
+
+    for (iMarker_Analyze = 0; iMarker_Analyze < nMarker_Analyze; iMarker_Analyze++) {
+      cout << "Surface "<< config->GetMarker_Analyze_TagBound(iMarker_Analyze) << ":" << endl;
+
+      if (nDim == 3) { if (si_units) cout << setw(20) << "Area (m^2): "; else cout << setw(20) << "Area (ft^2): "; }
+      else { if (si_units) cout << setw(20) << "Area (m): "; else cout << setw(20) << "Area (ft): "; }
+
+      if (si_units)      cout << setw(15) << fabs(Surface_Area_Total[iMarker_Analyze]);
+      else if (us_units) cout << setw(15) << fabs(Surface_Area_Total[iMarker_Analyze])*12.0*12.0;
+
+      cout << endl;
+
+      su2double MassFlow = config->GetSurface_MassFlow(iMarker_Analyze);
+      if (si_units)      cout << setw(20) << "Mf (kg/s): " << setw(15) << MassFlow;
+      else if (us_units) cout << setw(20) << "Mf (lbs/s): " << setw(15) << MassFlow;
```

Comment:
be kind on future-you and define the "20" and "15" somewhere, close to where you set the precision for example

Reply:
I have to admit the diff is a bit deceiving in that all these "output" lines are changed by me which I didn't do. I will take a look on what I can do

Reply:
ah ok don't worry about it then